src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -46,10 +52,7 @@ interface VideoPlayerState {
   player: Maybe<VideoPlayerInterface>;
 }
 
-export class VideoPlayer extends React.Component<
-  VideoPlayerProps,
-  VideoPlayerState
-> {
+class VideoPlayer extends React.Component<VideoPlayerProps, VideoPlayerState> {
```

Comment:
What is now `VideoPlayerProps` should probably only be used internally and therefore not exported IMO